stock_picking_force_assign/README.rst

```diff
@@ -14,13 +14,13 @@ Auto Unreserve
     :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
     :alt: License: AGPL-3
 .. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fstock--logistics--workflow-lightgray.png?logo=github
-    :target: https://github.com/OCA/stock-logistics-workflow/tree/12.0/stock_picking_force_assign
+    :target: https://github.com/OCA/stock-logistics-workflow/tree/14.0/stock_picking_force_assign
```

Comment:
this looks wrong

Reply:
Gets me every time. Fixed, and also pushed missing xml file.